File: 2057. Smallest Index With Equal Value/solution.js
/**
 * @param {number[]} nums
 * @return {number}
 */
 var smallestEqual = function(nums) {
  for (let index = 0; index < nums.length; index++){
    let num = nums[index];
    if (index % 10 === num){
      return index;
    }
  }
  return -1;
};

File: 2057. Smallest Index With Equal Value/solution.cpp
class Solution {
public:
  int smallestEqual(vector<int>& nums) {
    for (int index = 0; index < nums.size(); index++){
      int num = nums[index];
      if (index % 10 == num){
        return index;
      }
    }
    return -1;
  }
};